src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -54,8 +53,8 @@ class NexusPublishPlugin : Plugin<Project> {
             "Plugin must be applied to the root project but was applied to ${project.path}"
         }
 
-        require(GradleVersion.current() >= GradleVersion.version("6.0")) {
-            "The plugin requires Gradle version 6.0+"
+        require(GradleVersion.current() >= GradleVersion.version("6.2")) {
```

Comment:
If only we had some stats of Gradle version distributions out in the wild, like Android does: https://www.androidauthority.com/wp-content/uploads/2023/06/Android-version-distribution-statistics-May-2023-988w-675h.jpg.webp

It would be a much easier to say "minimum x".

Reply:
I asked about that one friendly Gradle engineer ;-). We will see.